sdks/java/expansion-service/src/test/java/org/apache/beam/sdk/expansion/service/ExpansionServiceSchemaTransformProviderTest.java

```diff
@@ -70,10 +70,10 @@ public class ExpansionServiceSchemaTransformProviderTest {
 
   private static final Schema TEST_SCHEMATRANSFORM_CONFIG_SCHEMA =
       Schema.of(
-          Field.of("str1", FieldType.STRING),
-          Field.of("str2", FieldType.STRING),
           Field.of("int1", FieldType.INT32),
-          Field.of("int2", FieldType.INT32));
+          Field.of("int2", FieldType.INT32),
+          Field.of("str1", FieldType.STRING),
+          Field.of("str2", FieldType.STRING));
```

Comment:
Schema needs to be in alphabetical order because tests use `TypedSchemaTransformProvider`, which returns a config schema with fields that are in alphabetical order. Recent commit that fixes encoding positions makes this required or else we end up with encoding error.